### PR TITLE
note addition

### PR DESCRIPTION
**Pull request by joinimran**
As per user recommendations and the support case opened with Microsoft, enabling a port is not possible with Firewall CSP.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2295

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/mdm/firewall-csp.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ Firewall rules in the FirewallRules section must be wrapped in an Atomic block i
 
 For detailed information on some of the fields below see [[MS-FASP]: Firewall and Advanced Security Protocol documentation](https://msdn.microsoft.com/library/mt620101.aspx).
 
+> [!NOTE]
+> The capability of enabling a rule in Firewall that will enable a port such as TCP 3389 using custom policies is not currently possible using a CSP.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/firewall-csp.md:24]:
```suggestion
> Enabling a rule in Firewall that will allow a port such as TCP 3389 to use custom policies is not currently possible using a CSP.
```

**Comment by ManikaDhiman** [comment]
I have reached out the product team to review the note. I will update this PR as soon as I hear from them. Thanks!

**Comment by e0i** [comment]
@ManikaDhiman Has there been any update for this from the product team? Please let us know how to proceed. Thank you.

**Comment by ManikaDhiman** [comment]
Hi @e0i, this is still under active review/discussion. I will update the status as soon as I hear more. Thanks!

**Comment by ManikaDhiman** [comment]
Hi @e0i, Per the product team, the statement in the comment is not true. They could test a syncml to add a rule that would allow TCP port 3389 (both local and remote port worked).

Additionally, Intune has support to be able to configure firewall rules via an endpoint protection device configuration profile as well as through a firewall endpoint security profile.

I am closing this PR as the note added here is not applicable/required. Thanks!

**Pull request closed without merging**